test/dotnet-test.UnitTests/GivenATestCommand.cs

```diff
@@ -14,10 +14,13 @@ namespace Microsoft.Dotnet.Tools.Test.Tests
     public class GivenATestCommand
     {
         private static readonly string ProjectJsonPath = Path.Combine(
-                AppContext.BaseDirectory,
+                Directory.GetCurrentDirectory(),
```

Comment:
Doesn't looks like this test project copied it in - I could do that. That said, the only bit this code does is check the existence of the project.json (and fails if it can't find it). So having the test sources doesn't help a great deal.